### ISSUE TITLE
Reset binding

### ISSUE THREAD

**Issue by bfred-it** [joshbernfeld/Transmission-Interface-Binder#4]
It seems that even after TIB is stopped, the binding remains. Is there a way to remove it altogether?

This is in the readme but it doesn't say how it can be solved:

> If my VPN disconnects while Transmission is open and the Interface Binder is running, will traffic revert to my default network?

> No. When Transmission first starts it will bind to the IP address of your selected interface. If that IP address and interface disappear, it will stay bound to it but no traffic will pass through. You will see your seeders disconnect and any outgoing connections dropped.

**Comment by joshbernfeld** [comment]
I purposefully left this functionality in because if the VPN connection drops we don't want it to revert to the default network.

I believe what you are asking for is if you hit the stop button on the interface binder GUI, you would like it to revert to the local network. It should normally do this after you manually restart the application. If it doesn't you can try clearing the bind IP property in the Transmission config file with the following terminal command.

`defaults delete ~/Library/Preferences/org.m0k.transmission BindAddressIPv4`

**Comment by fongd** [comment]
I figured this out the hard way when I realized none of my torrents were downloading anymore. I'd love to see this functionality more explicitly implemented in a future version, either as a button that allows one to manually remove the binding that's added or some other indication of how to do it (e.g., a note in the dialog). As it stands now, it's not obvious.

**Comment by joshbernfeld** [comment]
It may be better once the interface binder is disabled with the button, to remove the binding address, then leave it up to the user to restart transmission.

**Comment by fongd** [comment]
Yeah I like that functionality. Would also be nice to have an option to confirm quitting the app. I wasn't sure whether closing the window and quitting the app were the same thing when I started using this.
